### ISSUE TITLE
Ignoring camel-snake-kebab

### ISSUE THREAD

**Issue by dotemacs** [luminus-framework/luminus#152]
Hi @yogthos 

using conman in my own app and I love it how it wraps the SQL statements into Clojure functions.

Got the idea to flip some of the results that the SQL returns from snake case to kebab case. Did some googling and saw your write up on this:

https://github.com/luminus-framework/luminus/blob/master/resources/md/database.md#massaging-key-names-from-sql-to-clojure-style

I'm wondering why did you opt to write your own **->kebab-case-keyword** and **transform-keys**, when you could have just borrowed them from [camel-snake-kebab](https://github.com/qerub/camel-snake-kebab). That way the whole code snipped could look like this:

```clojure
(ns <<myapp.>>.db.core
  (:require ...
            [camel-snake-kebab.extras :refer [transform-keys]]
            [camel-snake-kebab.core :refer [->kebab-case-keyword]]))

(defn result-one-snake->kebab
  [this result options]
  (->> (hugsql.adapter/result-one this result options)
       (transform-keys ->kebab-case-keyword)))

(defn result-many-snake->kebab
  [this result options]
  (->> (hugsql.adapter/result-many this result options)
       (map #(transform-keys ->kebab-case-keyword %))))

(defmethod hugsql.core/hugsql-result-fn :1 [sym]
  'yuggoth.db.core/result-one-snake->kebab)

(defmethod hugsql.core/hugsql-result-fn :* [sym]
  'yuggoth.db.core/result-many-snake->kebab)
```

Did you just not want to add camel-snake-kebab, as it's one more dependency? Or...?

If you're not opposed to, I'd happily contribute the above to the documentation. But before I create the PR, let me know your thoughts. 

Thanks

**Comment by yogthos** [comment]
Yeah, I originally didn't want to introduce extra dependencies, but on a second thought it does look cleaner using camel-snake-kebab. If you'd like to PR the change, I think it's a good idea.

**Comment by dotemacs** [comment]
Cool, I'll do it in a few hours!

**Comment by dotemacs** [comment]
Resolved in the PR #153.

Closing.
